config/module/get_test.go

```diff
@@ -16,11 +16,26 @@ import (
 	version "github.com/hashicorp/go-version"
 )
 
-// map of module names and version for test module.
-// only one version for now, as we only lookup latest from the registry
-var testMods = map[string]string{
-	"registry/foo/bar": "0.2.3",
-	"registry/foo/baz": "1.10.0",
+// Map of module names and location of test modules.
+// Only one version for now, as we only lookup latest from the registry.
+type testMod struct {
+	location string
+	version  string
+}
+
+var testMods = map[string]testMod{
+	"registry/foo/bar": {
+		location: "file:///download/registry/foo/bar/0.2.3//*?archive=tar.gz",
```

Comment:
I forgot the comment explaining the optional scheme prefix for these tests. (this fixture is likely to evolve in the near future anyway)